src/plugins/cli-error-language.tsx

```diff
@@ -0,0 +1,13 @@
+module.exports = function(Prism) {
+  // common error messages
+  Prism.languages.error = {
+    error: /.*/
+  };
+
+  // amplify cli output messages
+  Prism.languages['amplify-cli-output'] = {
+    success: /^✔|✅.*/,
```

Comment:
In my local testing these regex need the global and multiline flags are needed here, also in some cases it seems like there are white spaces that appear before these markers.  For me a regex like this `/(^\s*✔|^\s*✅).*/gm` seems to be working.

Reply:
@jacoblogan modified the regex to consider more use cases and also bought the branch up to speed.